finance/proposal-calls/cycle3/bounties.md

```diff
@@ -0,0 +1,53 @@
+### Addressing long-standing bugs and feature requests by putting a monetary bounty
+
+### Project Team
+This project will put bounties on specific issues, that can be collecrted by anyone (which includes people without previous involvement in astropy) who writes a PR that closes that issue.
+
+The progam is organized by: Moritz Guenther (@hamogu), Erik Tollerud (@eteq) your..name..here..
+
+### Project Description
+Address long-standing bugs or missing features that volunteers don't address.
+
+### Project / Work
+The idea to use funding is to address long-standing issues and bugs, that apparently no volunteer has time or interest to take on. So, we could put a bounty in dollars on those issues, which gets paid to whoever gets a PR merged that successfully closes that issue.
+
+This concept is simple in principle, but raises a number of questions about the implementation. Who decides which issues are old enough to say "without money, this will never be done"? How much money do we put on each issue and which issues are important enough to pay for? Does the money actually motivate people to address things, or are those issues and bugs still open for other reasons (e.g. there is no known solution that's technically feasible)? Is there social pressure on reviewers to accept half-finished PRs because someone "needs the money for a living"? Will there even competition and wasted effort because suddenly several developers jump on the same issues?
+
+We believe that there is no way to design a perfect process in advance, we'll just have to take the plunge and try it out. Thus, we propose to put a limited amount of money (see budget below) into "issue bounties".
+We'll in particular see if the community is interested in earning the bug bounties and what amount of money is required to make this work.
+
+To this end we will select the following issues, which are over 3 years old, look reasonably self-contained, and could be addressed (i.e. don't need work on several other issues before they can be addressed) by a variety of developers (i.e. they may be complex in programming, but don't require APEs or decision on API that only the maintainers can make):
+
+- https://github.com/astropy/astropy/issues/4112 - 300 $ + 75 $ for review
+- https://github.com/astropy/astropy/issues/3720 - 300 $+ 75 $ for review
+- https://github.com/astropy/astropy/issues/4121 - 1600 $ + 400 $ for review
+- https://github.com/astropy/astropy/issues/4610 - 1600 $ + 400 $ for review
+- https://github.com/astropy/astropy/issues/1803 - 3000 $ + 750 $ for review
```

Comment:
I haven't been following this closely. Is this the *max possible* or flat rate? Seems like a lot of money for one issue, and what if it turns out to be really simple fix? 

My comment is in general, not just about that one issue.

Reply:
I'm suggesting that for this particular issue. That's based on my best guess how long it might take. The point here would be to offer fixed-price tasks. If someone finds an easy fix, that's great. However, all those issue have been open for years. So, if there was an obvious, simple fix, I would assume someone would have found it by now. I looked at the issues and tried to make a realistic estimate of how long it might take. 

That's the weak point here - I may not have seen how easy that it. But that's part of this proposal, to see if that works or not. If you have a different suggestion or a better idea to come up with these numbers, please let me know.

If you are opposed to fixed-price offers in general, please downvote.